boards/deneyapmini.json

```diff
@@ -37,11 +35,9 @@
     "flash_size": "4MB",
     "maximum_ram_size": 327680,
     "maximum_size": 4194304,
-    "use_1200bps_touch": true,
-    "wait_for_upload_port": true,
```

Comment:
https://github.com/espressif/arduino-esp32/blob/master/boards.txt#L17979
boards.txt states that this board requires 1200bps touch by default, is it wrong?

Reply:
> https://github.com/espressif/arduino-esp32/blob/master/boards.txt#L17979 boards.txt states that this board requires 1200bps touch by default, is it wrong?

You are right. I will update it.

Reply:
> You are right. I will update it.

The same with `deneyapminiv2`, seems like 1200bps is required.

Reply:
Fix `use_1200bps_touch` and `wait_for_upload_port` parameters of Deneyap Mini v2, Deneyap Mini and Deneyap Kart 1A v2.

Reply:
We have a [PR](https://github.com/espressif/arduino-esp32/pull/8046) for update `use_1200bps_touch` and `wait_for_upload_port` for Deneyap Kart 1A v2.